src/gcodeExport.cpp

```diff
@@ -711,8 +727,17 @@ void GCodeExport::writeUnretractionAndPrime()
         }
         else
         {
-            current_e_value += extruder_attr[current_extruder].retraction_e_amount_current;
-            *output_stream << "G1 F" << PrecisionedDouble{1, extruder_attr[current_extruder].last_retraction_prime_speed * 60} << " " << extruder_attr[current_extruder].extruderCharacter << PrecisionedDouble{5, current_e_value} << new_line;
+            *output_stream << "G1 F" << PrecisionedDouble{1, extruder_attr[current_extruder].last_retraction_prime_speed * 60} << " " << extruder_attr[current_extruder].extruderCharacter;
+            if (relative_extrusion)
```

Comment:
I think you should refactor this the same as above.
Introduce an `output_e` and don't wriote to the `output_stream` inside the branches of an if-statement.